packages/p2p/src/components/advertiser-page/block-user/block-user-overlay.scss

```diff
@@ -0,0 +1,34 @@
+$header-height-offset: 23.8rem;
+$mobile-header-height-offset: 17.1rem;
+
+.block-user-overlay {
+    align-items: center;
+    background-color: var(--general-main-1);
+    border: 2px solid var(--general-section-1);
+    border-radius: 6px;
+    display: flex;
+    flex-direction: column;
+    height: calc(100vh - #{$header-height-offset} - 5.8rem);
+    justify-content: center;
+    position: fixed;
+    opacity: 0.9;
+    top: #{$header-height-offset};
+    width: 937px;
```

Comment:
Use `rem` for width and height

Reply:
I forgot to refactor this, will refactor it ASAP